DataManagementSystem/scripts/dirac-dms-find-lfns.py

```diff
@@ -10,6 +10,7 @@
 
   from DIRAC.Core.Base import Script
 
+  Script.registerSwitch( '', 'Path=', '    Path to search for' )
```

Comment:
In principle this is not necessary because Path is a valid metadata key. Historically this was not the case, so it is a separate argument in the call now. I wanted to get rid of it in fact. You can simply do:
    dirac-dms-find-lfns Path=/lhcb/user/a/atsareg SE=IN2P3-disk